app/cardrecord.js

```diff
@@ -28,23 +28,27 @@ class CardRecorder extends MyTrello {
                 self.deleteCurrentComment(comments).then(function(resp) {
                     var updateActions = _.filter(card.actions, {type: "updateCard"});
                     var createAction = _.filter(card.actions, {type: "createCard"});
+                    var now = moment();
                     var hasMoved = false;
+                    var daysSinceUpdate = false;
                     if(typeof updateActions !== "undefined"){
                       hasMoved = self.hasMovedCheck(updateActions);
+                      daysSinceUpdate = now.diff(moment(updateActions[0].date), 'days');
                     }
+                    var totDays = (comments !== "undefined")? self.calcTotalDays(comments, now) : 0;
+                    console.log("total days:"+totDays);
                     var lastMove = self.findLastMoveDateFromComments({commentList: comments, "actionList": updateActions, "createActionDate": createAction.date});
```

Comment:
I'm curious if instead of reading the date from a comment, could you use the date from the last move?  That way you're not having to parse text.  Here, the `hasMoved` var is either false or a list of move actions.  If you find the most recent move action, that should be the `lastMove` date.